_elm_package.sh

```diff
@@ -169,6 +169,9 @@ _elm_package()
         if [[ $cur == -* ]]; then
             _suggest_package_diff_flags $cur
             return 0
+        else
+            _suggest_package_name $cur
```

Comment:
I think this might actually be wrong -- what gets suggested when you have `elm-package diff elm-lang/core 1.0.0 [TAB]`? What do you get suggested at `[TAB]`?

Reply:
Ah yes, package name get suggested after first version number

Reply:
But before there were no suggestions at all, neither for package name nor for version numbers. E.g. `elm-package install elm-lang/core [TAB]` will also suggest name packages.

Reply:
maybe you can figure out howto fix that as part of this pr?

Reply:
What do you think about checking the word index but excluding indexes for flags? It should always be the same number.
E.g.
In `elm-package install package/name` `package/name` will have index `2`. In `elm-package install --yes package/name` `package/name` will have index `3` minus `1` for `--yes` tag which will be again `2`.

So if we check that current index minus number of flags equals to `2` than we do `_suggest_package_name $cur`. Otherwise do nothing.

Reply:
Or do you mean auto completion for version numbers?

Reply:
I've added the function, now it works as expected. What you you think?